src/Plugins/BotSharp.Plugin.AudioHandler/Functions/AudioProcessUtilities.cs

```diff
@@ -0,0 +1,68 @@
+using BotSharp.Plugin.AudioHandler.Enums;
+using NAudio;
+using NAudio.Wave;
+using NAudio.Wave.SampleProviders;
+
+namespace BotSharp.Plugin.AudioHandler.Functions;
+
+public class AudioProcessUtilities : IAudioProcessUtilities
+{
+    public AudioProcessUtilities()
+    {
+    }
+
+    public Stream ConvertMp3ToStream(string mp3FileName)
+    {
+        var fileStream = File.OpenRead(mp3FileName);
```

Comment:
Change to `using var fileStream = `

Reply:
I tested it on my local. Adding "using" statement at the beginning will result in a disposed "fileStream" if SampleRate is 16000. The value returned by this function will not contain any data.